### ISSUE TITLE
Reference postprocess_qa_predictions score method

### ISSUE THREAD

**Issue by LozanoAlvarezb** [huggingface/transformers#12399]
To the best of my knowledge literature on question answering computes the probability of a span by first computing the probability for a token to be the start token or the end token independently and then multiplying those probabilities. In contrast, the [postprocess_qa_predictions](https://github.com/huggingface/transformers/blob/57461ac0b4e4f7349c2437fcf8d4115014d6ceda/examples/pytorch/question-answering/utils_qa.py#L31) function computes the probability for an answer by first summing both logits, ranking them, and applying the softmax over the top n_best scores.

Is there any reference in the literature supporting this?
